app/femr/ui/controllers/TriageController.java

```diff
@@ -161,8 +161,11 @@ public Result indexPost(int id) {
 
 
         //create and save a new encounter
-        PatientEncounterItem patientEncounterItem =
-                populatePatientEncounterItem(viewModel.getChiefComplaint(), viewModel.getChiefComplaintsJSON(), viewModel.getWeeksPregnant(), currentUser, patientServiceResponse.getResponseObject().getId(), viewModel.getAgeClassification());
+        // PatientEncounterItem patientEncounterItem =
+                // populatePatientEncounterItem(viewModel.getChiefComplaint(), viewModel.getChiefComplaintsJSON(), currentUser, patientServiceResponse.getResponseObject().getId(), viewModel.getAgeClassification());
+        // ServiceResponse<PatientEncounterItem> patientEncounterServiceResponse = encounterService.createPatientEncounter(patientEncounterItem);
```

Comment:
Its great to comment out the previous code while working on new stuff, but I think once the new code is working, go ahead and delete the previous stuff. If we need to see what was there before, we can go back in the git history :)